Framework/Muon/src/MuonPreProcess.cpp

```diff
@@ -94,16 +101,27 @@ std::map<std::string, std::string> MuonPreProcess::validateInputs() {
     }
   }
 
-  // Checks for dead time table
+  // Checks for dead time table and time zero table
```

Comment:
`validateInputs` is starting to grow a bit long in the tooth
Can we split this down by moving line 104-125 into a private method like 
`std::map<std::string, std::string> validateTableInputs()`

This way future devs looking at the validation code know if something is wrong with table validation they only need to go to that method, rather than read the entire `validateInputs()`